input/system/selfhosted/otel_handler.go

```diff
@@ -15,6 +16,9 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+// A list of all OpenTelemetry servers that we have started to prevent
+var otelServers []string
```

Comment:
This may not work as expected because global state doesn't get reset on reloads - can we put this into a function somewhere (or a different state struct?)